src/languages/php.js

```diff
@@ -11,23 +11,101 @@ Category: common
  * @returns {LanguageDetail}
  * */
 export default function(hljs) {
+  const regex = hljs.regex;
   const VARIABLE = {
-    className: 'variable',
+    scope: 'variable',
     begin: '\\$+[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*' +
       // negative look-ahead tries to avoid matching patterns that are not
       // Perl at all like $ident$, @ident@, etc.
       `(?![A-Za-z0-9])(?![$])`
   };
+  const CONSTANT = {
+    variants: [
+      {
+        begin: [
+          /const\s+/,
```

Comment:
Please split to exclude the `\s+` from the keyword scope.

Reply:
done